### DISCUSSION TITLE
Github Campus: Local expert recommendation algorithm isn't quite exact

### DISCUSSION THREAD

**Discussion by TSStudio** [community/community#39913] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The local expert Github chose for me is surely near me, but **geographically**.  
I'm living in Jilin, China, and it's ture one in South Korea is close to me. But different countries or languages should be considered. From my view, an expert from Beijing or Shanghai or even anywhere in the country is "more local".

**Comment by ossanna16** [comment]
Hi there @TSStudio and welcome to our community! Thank you for the feedback!
